lnwire/channel_reestablish.go

```diff
@@ -70,6 +84,11 @@ type ChannelReestablish struct {
 	// TODO(roasbeef): rename to verification nonce
 	LocalNonce *Musig2Nonce
 
+	// DynHeight is an optional field that stores the dynamic commitment
+	// negotiation height that is incremented upon successful completion of
+	// a dynamic commitment negotiation
+	DynHeight *DynHeight
```

Comment:
Could be an optional?

Reply:
Fixed. Should I also do the same thing to the LocalNonce field even though it's unrelated to dyncomms or leave it heterogeneous for now?